test/JDBC/expected/sp_changedbowner-vu-verify.out

```diff
@@ -0,0 +1,404 @@
+
+-- tsql
+-- sp_changedbowner: this is a subset of the tests for C:\Babelfish\code\work\chgdbowner\alter_authorization_change_db_owner
+use master
+go
+alter login new_OWNER_login_sp with password='12345678'
+go
+alter login new_OWNER_login_sp2 with password='12345678'
+go
+alter login [new_OWNER_login_sp3] with password='12345678'
+go
+alter login [new_OWNER_login_sp 4] with password='12345678'
+go
+set quoted_identifier on
+go
+alter login "new_OWNER_login_sp5" with password='12345678'
+go
+alter login "new_OWNER_login_sp 6" with password='12345678'
+go
+set quoted_identifier off
+go
+
+-- system database ownership cannot be changed
+use master
+go
+execute sp_changedbowner 'new_owner_login_sp'
+go
+~~ERROR (Code: 50000)~~
+
+~~ERROR (Message: Cannot change the owner of the master, model, tempdb or distribution database.)~~
+
+use tempdb
+go
+execute sp_changedbowner 'new_owner_login_sp'
+go
+~~ERROR (Code: 50000)~~
+
+~~ERROR (Message: Cannot change the owner of the master, model, tempdb or distribution database.)~~
+
+
+-- except for msdb
+use msdb
+go
+declare @orig_owner sysname 
+select @orig_owner = suser_sname(sid) from sysdatabases where name = 'msdb'
+execute sp_changedbowner new_owner_login_sp
+select name, suser_sname(sid) from sysdatabases where name = 'msdb'
+execute sp_changedbowner @orig_owner
+select name, suser_sname(sid) from sysdatabases where name = 'msdb'
+go
+~~START~~
+text#!#nvarchar
+msdb#!#new_OWNER_login_sp
+~~END~~
+
+~~START~~
+text#!#nvarchar
+msdb#!#jdbc_user
```

Comment:
test could fail in our internal repo because default owner may not be `jdbc_user` anymore when we run it on internal gitfarm

Reply:
will adjust